### PR TITLE
Feature/updateUserModal

### PR DESCRIPTION
**Pull request by Retrofitt**
## Description

As per Jason's request, Implement modals when clicking on users to update and manage within the UserManagement.js component.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

## Loom Video
https://www.loom.com/share/c77bbf44b0ec4308a3d25ed282c074b8

### CODE REVIEW

File: src/components/pages/UserManagement/RenderUserManagement.js [hunk 1]
```diff
@@ -326,37 +344,32 @@ function RenderUserManagement(props) {
...
-                type="primary"
-                id="submitChanges"
-                disabled={isDisabled}
-                onClick={() => updateUser(user)}
-              >
-                Update user info
-              </Button>
-              <Button id="cancelChanges" onClick={cancelChanges}>
-                Revert changes
-              </Button>
```
Comment (Retrofitt) [inline src/components/pages/UserManagement/RenderUserManagement.js:339]: These Buttons are now found within the User Management Modal

**Review by Pluviio — APPROVED** [review]
Seems like you took what Jason said and hit it directly. While watching the video it looks more seamless and clean. Easier to navigate through

**Review by nhamilton1 — APPROVED** [review]
Great video, the new design looks so clean, fantastic work!

**Review by cyberkade — APPROVED** [review]
I love the style changes to this component. A modal makes for a more direct, organized approach to the user management. Let's get this merged!

**Pull request merged**